### PR TITLE
TorchAgent doc reformat

### PR DESCRIPTION
**Pull request by stephenroller**
Reformat TorchAgent docs.

I propose this as our standard format.

### CODE REVIEW

File: parlai/core/torch_generator_agent.py [hunk 1]
```diff
@@ -6,11 +6,8 @@
 
 
 """
-**BETA**: This module is in beta. Feedback is most welcome, and the API may change
-underneath you.
-
-Generic Pytorch-based Generator agent. Implements quite a bit of boilerplate,
-including Beam search.
+Generic PyTorch-based Generator agent. Implements quite a bit of boilerplate,
```
Comment (klshuster) [inline parlai/core/torch_generator_agent.py]: no longer in beta?

File: parlai/core/torch_agent.py [hunk 2]
```diff
@@ -44,51 +44,41 @@ class Batch(AttrDict):
     with additional fields if they would like, though we recommend calling the
     parent function to set up these fields as a base.
 
-    .. py:attribute:: text_vec
-
+    :param text_vec:
```
Comment (klshuster) [inline parlai/core/torch_agent.py]: any reason we still use `:attribute:` below in torch generator but not here?

File: parlai/core/torch_agent.py [hunk 3]
```diff
@@ -130,20 +118,31 @@ def __init__(self, text=None, text_candidates=None, **kwargs):
...
-                         'person_tokens' set to True for this to be added
-    :param p1_token:     token indicating 'person 2'; opt must have
-                         'person_tokens' set to True for this to be added
-    :param dict_agent    DictionaryAgent object for tokenizing the history
+    History handles tracking the dialogue history/state over the course of an
+    episode.
+
+    History may also be used to track the history of any field.
+
+    :param field:
```
Comment (klshuster) [inline parlai/core/torch_agent.py]: same comment here with `attribute`

File: parlai/core/torch_agent.py (hunk 2 above)
Comment (stephenroller) [reply parlai/core/torch_agent.py]: I will want to update that, good catch

**Pull request merged**